src/components/flyout/deviceReconfigureFlyout.js

```diff
@@ -190,8 +190,10 @@ class DeviceReconfigureFlyout extends React.Component {
     this.state.commonConfiguration.forEach(item => {
       reportedProps[item.label] = item.value;
     });
+    let jobName = this.state.jobInputValue ;
```

Comment:
`const jobName = this.state.jobInputValue.replace(/\W/g, "");`

Note the change in the regex. You should also pull this regex into a utility file that is imported across all the other places it is used.

Reply:
I think, (/[\W_]/g, "") this would be the right regex code, because we have to eliminate the _ and  [] also.